sig-cf/README.md

```diff
@@ -0,0 +1,29 @@
+# SIG: CF
+
+This SIG focuses on the current Cloud Foundry (CF) clouddriver provider for Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Last Thursday of each month, at 1 PM EST / 10AM PST
+  
+  [Agenda](https://docs.google.com/document/d/1TouFx5ykNCcZxNSD2JhUiRliZDXwxZ6sx239rigYdxw/edit?usp=sharing)
+
+## Leadership
+
+* Zach Smith - Armory (https://github.com/zachsmith1)
+* Ryan Pei - Armory (https://github.com/ryanpei)
+
+## Contact
+
+* [Slack](https://spinnakerteam.slack.com/archives/C4ENF2E1W)
```

Comment:
The convention for the other SIGs is to have the channel be called `#sig-{name}`; would it make sense to use that here to for discoverability?

Reply:
good idea!